### PR TITLE
add group test

### PR DESCRIPTION
**Pull request by SOLenG**

### CODE REVIEW

File: GroupBundle/Tests/Document/GroupTest.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace OpenOrchestra\GroupBundle\Tests\Document;
+
+use OpenOrchestra\GroupBundle\Document\Group;
+use \Phake;
```
Comment (nicolasThal) [inline GroupBundle/Tests/Document/GroupTest.php]: type `use Phake` instead

File: GroupBundle/Tests/Document/GroupTest.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    protected $label;
+    protected $group;
+    protected $site;
+
+    /**
+     * set Up
+     */
+    public function setUp()
+    {
+        $this->group = new Group();
```
Comment (nicolasThal) [inline GroupBundle/Tests/Document/GroupTest.php]: I prefer to create the object that we run the test on at the end of the setUp method.

Moreover, i like to leave one blank line before his creation

File: GroupBundle/Tests/Document/GroupTest.php [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        $this->site = Phake::mock('OpenOrchestra\ModelInterface\Model\ReadSiteInterface');
+        $this->labels = Phake::mock('Doctrine\Common\Collections\ArrayCollection');
+        $this->label = Phake::mock('OpenOrchestra\ModelInterface\Model\TranslatedValueInterface');
+        Phake::when($this->label)->getLanguage()->thenReturn('en');
+    }
+
+    /**
+     * test site
+     */
+    public function testSite(){
```
Comment (nicolasThal) [inline GroupBundle/Tests/Document/GroupTest.php]: cs

File: GroupBundle/Tests/Document/GroupTest.php [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        $this->assertSame(null, $this->group->getSite());
+        $this->group->setSite($this->site);
+        $this->isTrue($this->site == $this->group->getSite());
```
Comment (nicolasThal) [inline GroupBundle/Tests/Document/GroupTest.php]: $this->assertSame($this->site, $this->group->getSite());

File: GroupBundle/Tests/Document/GroupTest.php [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    }
+
+    /**
+     * test getTranslatedProperties
+     */
+    public function testGetTranslatedProperties()
+    {
+        $this->assertSame(['getLabels'], $this->group->getTranslatedProperties());
+    }
+
```
Comment (nicolasThal) [inline GroupBundle/Tests/Document/GroupTest.php]: suppress blank lines

File: GroupBundle/Tests/Document/GroupTest.php [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+        $this->group->setSite($this->site);
+        $this->isTrue($this->site == $this->group->getSite());
+    }
+
+    /**
+     * test Labels
+     */
+    public function testLabels()
+    {
+        $this->isTrue($this->labels == $this->group->getLabels());
```
Comment (nicolasThal) [inline GroupBundle/Tests/Document/GroupTest.php]: never use isTrue

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**